misc/tokens.pl

```diff
@@ -220,3 +220,4 @@ sub normalize_name {
 0 0 0 0 0 x-forwarded-for
 0 0 0 0 0 x-traffic
 0 0 0 0 0 cache-digest
+0 0 0 0 0 compress-hint
```

Comment:
Thank you for the link. I think we should retain `x-`, considering the fact that the header is a signal used _internally_ (we drop the header in H2O). The headers that fall into the criteria are not subject to the discussion of deprecation, quoting from RFC 7231:
```
   Authors of specifications defining new fields are advised to keep the
   name as short as practical and not to prefix the name with "X-"
   unless the header field will never be used on the Internet.
```